### PR TITLE
Port libAfterImage on Win64

### PR DESCRIPTION
**Pull request by bellenot**
Add `libAfterImage/CMakeLists.txt` to facilitate the port of libAfterImage on Win64

### CODE REVIEW

**Review by oshadura — COMMENTED** [review]
@bellenot cool job! I was thinking what if we try to move libAfterImage to 'builtins' directory: `https://github.com/root-project/root/tree/master/builtins` (we can try to do this in the next iteration if you want?)

**Comment by bellenot** [comment]
> @bellenot cool job! I was thinking what if we try to move libAfterImage to 'builtins' directory: `https://github.com/root-project/root/tree/master/builtins` (we can try to do this in the next iteration if you want?)

Thanks @oshadura ! I think this question is more for @couet who is responsible for the graphics

**Review by couet — APPROVED** [review]

**Comment by couet** [comment]
@oshadura What does it imply to move it to "builtins" ? If it does not change anything to the current status it is fine (ie if is is only a administration issue). This library is the corner stone for the binary output of binary images and need to be built when installing ROOT as the version we have is "our" (many mods in it for us) and we cannot rely on an external build one.

File: graf2d/asimage/src/libAfterImage/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# libAferImage CMakeLists.txt
+
+PROJECT(AFTERIMAGE)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.0)
```
Comment (oshadura) [inline graf2d/asimage/src/libAfterImage/CMakeLists.txt:4]: Quick question, should we put here the minimum CMake version as it is in ROOT?
e.g. 
```
if(MSVC)
  # required for the following feature & bug fix:
  # 3.15: Added $<REMOVE_DUPLICATES:list> generator expression
  # 3.16: Bug fix with CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS: the auto-generated exports
  #       are now updated only when the object files providing the symbols are updated
  cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
else()
  cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
endif()
```

**Review by oshadura — APPROVED** [review]
One small comment, otherwise we are good! About moving it to builtins I  think it will be more convenient to start a separate thread with @couet :)

File: graf2d/asimage/src/libAfterImage/CMakeLists.txt (hunk 1 above)
Comment (bellenot) [reply graf2d/asimage/src/libAfterImage/CMakeLists.txt:4]: Oh right, I don't even know why there is a minimal version here, but I agree, I'll set it to 3.16, thanks @oshadura !

**Pull request merged**